priv/dev_server.exs

```diff
@@ -0,0 +1,128 @@
+Mix.install([
+  {:bandit, "~> 1.1"},
+  {:websock_adapter, "~> 0.5"},
+  {:file_system, "~> 0.2.10 or ~> 1.0"}
+])
+
+defmodule Reloader do
+  def init(_args) do
+    FileSystem.subscribe(:live_reload_file_monitor)
+    {:ok, []}
+  end
+
+  def handle_in({"ping", [opcode: :text]}, state) do
+    {:reply, :ok, {:text, "pong"}, state}
+  end
+
+  def handle_info({:file_event, _, {path, [:modified, :closed]}}, state) do
+    if String.contains?(path, "#{__DIR__}/static") do
+      Process.send_after(self(), :reload, 50)
+      {:push, {:text, "reload"}, state}
```

Comment:
Why do we reload twice? What if compiling takes longer than 50ms? Maybe we could send a message from recompiler to reloader when the compilation is finished, instead of subscribing to `live_reload_file_monitor` in both processes?